pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -109,25 +109,39 @@ func validateTargetThresholds(targetThresholds api.ResourceThresholds) bool {
 	return true
 }
 
-func classifyNodes(npm NodePodsMap, thresholds api.ResourceThresholds, targetThresholds api.ResourceThresholds) ([]NodeUsageMap, []NodeUsageMap, []NodeUsageMap) {
-	lowNodes, targetNodes, otherNodes := []NodeUsageMap{}, []NodeUsageMap{}, []NodeUsageMap{}
+// isNodeUschedulable checks if the node is unschedulable. This is helper function to check only in case of
+// underutilized node so that they won't be accounted for.
+func isNodeUschedulable(node *v1.Node) bool {
```

Comment:
move it to https://github.com/kubernetes-incubator/descheduler/blob/master/pkg/descheduler/node/node.go?

Reply:
Makes sense. Moved it there.